Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,110 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+ /* General */
+
+body{
+    font-family: Helvetica, sans-serif;
+}
+
+/* Topbar */
+.topbar{
+    height:70px;
+    width:100%;
+}
+
+.logo{
+    margin-left:20px;
+}
+/* big image section */
+.top-container{
+    display: flex;
+    flex-direction: column; 
+    justify-content: center;
+    align-items: center;
+    background-image: url('../images/flowershop.jpg');
+    background-position: center center ;
+    background-size:cover;
+    background-repeat: no-repeat;
+    height: 720px;
+
+}
+
+.overlay{
+    position: absolute;
+    background-color: black;
+    opacity:50%;
+    height:720px;
+    width: 100%;
+    z-index: 1;
+}
+
+
+.website-title{
+    z-index: 2;
+    color:white;
+    font-size: 3.5em;
+    letter-spacing: 3px;
+    text-shadow:2px 2px black;
+}
+
+.learnmore-btn{
+    z-index: 2;
+    margin-top:20px;
+    border-radius: 30px;
+    height: 60px;
+    width: 200px;
+    font-size: 2em;
+    text-align: center;
+    color:white;
+    background-color: #99f03d;
+    padding: 5px;
+    border-style:none;
+}
+/* service section */
+
+.bottom-container{
+    display: grid;
+    grid-template-columns: repeat(2,auto);
+    height: fit-content;
+    background-color:#cee7b4;
+    padding: 100px auto;
+    text-align:center;
+}
+.services{
+    margin: 45px;
+    text-align: center;
+}
+
+.service-description{
+    display: inline-block;
+    margin: 15px 80px;
+    line-height: 1.5;
+    font-weight: lighter;
+}
+
+/* Responsiveness */
+
```

Comment:
Great use of the grid system to style the four icons, with this in mind your use of @media is very smart. Great work!